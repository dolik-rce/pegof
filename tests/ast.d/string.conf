input ast.d/string.peg
ast
