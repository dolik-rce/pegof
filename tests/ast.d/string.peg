X <- "double quoted" 'single quoted' "\n'\"\\	"
