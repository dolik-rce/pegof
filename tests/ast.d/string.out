GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          STRING double quoted
        TERM
          STRING single quoted
        TERM
          STRING _\n\'\"\\\t
